script/hassfest/coverage.py

```diff
@@ -19,6 +19,7 @@
     "recorder.py",
     "scene.py",
 )
+FORCE_COVERAGE = ("gold", "platinum")
```

Comment:
If we add `internal` to the list, we have:
- browser
- downloader
- folder_watcher
- kitchen_sink
- otp
- ping
- recorder
- stream
- time_date
- zeroconf